src/commands/executeFunction.ts

```diff
@@ -79,6 +77,11 @@ export async function executeFunction(context: IActionContext, node?: FunctionTr
             if (!client && parseError(error).errorType === 'ECONNREFUSED') {
                 context.errorHandling.suppressReportIssue = true;
                 throw new Error(localize('failedToConnect', 'Failed to connect. Make sure your project is [running locally](https://aka.ms/AA76v2d).'));
+            } else if (parseError(error).message === 'Unexpected status code: 400') {
```

Comment:
Is there something else you can use to identify the error? Maybe `errorType` or something? Messages aren't always reliable because depending on the language, it might be different. Wording will also change sometimes.

Reply:
Since we're now using `parseError(error)` in two places, can you store the value in a variable and access it through that instead?

Something like:
```javascript
const pError = parseError(error);

if (pError.errorType ....
if (pError.message ....
```